pkg/volumeclaimbinder/persistent_volume_claim_binder.go

```diff
@@ -158,6 +158,14 @@ func syncVolume(volumeIndex *persistentVolumeOrderedIndex, binderClient binderCl
 
 	// available volumes await a claim
 	case api.VolumeAvailable:
+		// TODO:  remove api.VolumePending phase altogether
```

Comment:
@bgrant0607 re: API changes - how breaking is removing a phase in your eyes?  Should we press to get this in before 1.0?